.github/CODEOWNERS

```
@@ -1,8 +1,2 @@
 # GLOBAL OWNER
-*   @kmpaul
-
-# CI OWNER
-/ci/   @andersy005 @kmpaul
-
-# CONTENT OWNER
-/content/   @jukent @kmpaul @ktyle @clyne @brian-rose
+*   @ProjectPythia/infrastructure @ProjectPythia/education
```

Should this be used instead?

```suggestion
*   @ProjectPythia/infrastructure

/content/ @ProjectPythia/infrastructure @ProjectPythia/education
```